node-deploy.azure.yaml

```diff
@@ -41,5 +41,5 @@ spec:
               cpu: #{cpu}#
               memory: #{memory}#
           env:
-            - name: LOGIN_URL
-              value: https://app.bcrumbs.net/account/login
+            name: LOGIN_URL
+            value: ${LOGIN_URL}
```

Comment:
I think this one should be "https://app.bcrumbs.net/account/login" not ${LOGIN_URL}, isn't it?